docker/package/model.py

```diff
@@ -117,6 +118,9 @@ def gen_spec_systemd_part(package):
                 + f"%{{buildroot}}/%{{_sysconfdir}}/default/{service_name}\n"
             )
             default_files += f"%{{_sysconfdir}}/default/{service_name}\n"
+        if systemd_unit.service_file.service.environment_files is not None:
```

Comment:
A small nitpick, this change should probably belong to `[#517] Support multiple environment files in systemd unit` commit